maven-plugin-api/src/main/java/org/apache/maven/plugin/descriptor/MojoDescriptor.java

```diff
@@ -51,9 +52,7 @@ public class MojoDescriptor extends ComponentDescriptor<Mojo> implements Cloneab
 
     private static final String DEFAULT_LANGUAGE = "java";
 
-    private List<Parameter> parameters;
-
-    private Map<String, Parameter> parameterMap;
+    private final ArrayList<Parameter> parameters;
```

Comment:
Can be `List` type?

Reply:
Can, you have to complain to @cstamas. I'd like to leave as-is to make backports as easy as possible.

Reply:
Ok I see a source commit 😄